src/components/feedback/Dialog/index.tsx

```diff
@@ -0,0 +1,36 @@
+import * as React from 'react';
+import styles from './styles.less';
+import classNames from 'classnames';
+
+type DialogProps = {
+    children: any;
+};
+
+type DialogContentProps = {
+    scrollable?: boolean;
+};
+
+export const Dialog: React.FC<DialogProps> = ({ children }) => {
+    return <div className={styles.container}>{children}</div>;
+};
+
+export const DialogTitle: React.FC<DialogProps> = ({ children }) => (
+    <div className={styles.dialogTitle}>{children}</div>
+);
+
+export const DialogContent: React.FC<DialogProps & DialogContentProps> = ({ children, scrollable }) => {
+    const contentClasses = classNames(styles.dialogContent, {
+        [styles.scrollable]: scrollable,
```

Comment:
Is the scrollable option something that should be controlled by the user of the component, or by the component it self? You don't actually know that it should be scrollable until you have tested if the contents fits within the dialog. Would it be better to check the scrollHeight/Width within useEffect and apply the scrollable styling if the content overflows?

Reply:
Discussed with Lotte, scroll is only used in special cases for mobile and is not needed for now. So I removed it.